src/lib/response-body-listener.ts

```diff
@@ -20,17 +20,22 @@ export class ResponseBodyListener {
       details.requestId,
     ) as any;
 
-    const decoder = new TextDecoder("utf-8");
+    // const decoder = new TextDecoder("utf-8");
     // const encoder = new TextEncoder();
 
-    let responseBody = "";
+    let responseBody = new Uint8Array();
```

Comment:
Why define responseBody here and then use the tmp variable below? Can we just define below?

Reply:
I think that's not possible since _responseBody_ is also used in _filter.onstop_.

The tmp variable is used to concatenate two Uint8 arrays. The alternative would be to implement a generic Uint8Array concatenate function and then get rid of the tmp variable.

Reply:
Got it. Let's keep it as is then.

Reply:
Got it, let's keep it as is then. Thanks!